src/main/java/se/iths/springbootgroupproject/services/MessageService.java

```diff
@@ -72,4 +71,13 @@ public Optional<Message> findById(Long id) {
     public List<Message> fidAllByUserId(Long userId) {
         return messageRepository.findAllByUserId(userId);
     }
+
+
+//    public Message translateMessage(Message message) {
+//        String translatedTitle = translationService.translateText(message.getMessageTitle());
+//        message.setMessageTitle(translatedTitle);
+//        String translatedBody = translationService.translateText(message.getMessageBody());
+//        message.setMessageBody(translatedBody);
+//        return message;
+//    }
```

Comment:
should we remove this if it is not needed?

Reply:
I kept it to use if I would find a solution to the language detection problem. But I don't think I want to work with it any more, so I think we can remove it.